.github/workflows/update-make-docs.yml

```diff
@@ -1,5 +1,6 @@
 name: Update `make docs` procedure
-permissions: {}
+permissions:
+  contents: read
```

Comment:
This workflow creates branches and opens PRs so may need `contents: write` and `pull-requests: write`

Reply:
@jdbaldry Could you review and approve the PR? Looks like CI is waiting for your approval
```
    Waiting on code owner review from jdbaldry.
```